src/sql/workbench/contrib/notebook/common/constants.ts

```diff
@@ -0,0 +1,7 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+import { localize } from 'vs/nls';
+
+export const promptReloadNotebooks = localize('promptReloadNotebooks', "The setting for how tables are displayed in text cells has been changed. Please reload Azure Data Studio or reopen current notebooks to enable it.");
```

Comment:
Update this to match the new behavior (reload will reload the current cell - which will lose all edits in progress)